### ISSUE TITLE
Can you add Fingerprint ?

### ISSUE THREAD

**Issue by deandeandeanmaillab** [Doko-Demo-Doa/react-native-sello#2]
Hi,

Will you be able to add fingerprint support? and delete pin codes?

Thanks!!!!

**Comment by Doko-Demo-Doa** [comment]
No, not now. Since the native library lacks of documents and the code itself is a mess. I will halt the development for now.
